reaper_test.go

```diff
@@ -408,6 +408,22 @@ func Test_NewReaper(t *testing.T) {
 				"TESTCONTAINERS_HUB_IMAGE_NAME_PREFIX": "registry.mycompany.com/mirror",
 			},
 		},
+		{
+			name: "Reaper with a custom listen address",
```

Comment:
suggestion: simple test names
```suggestion
			name: "custom-address",
```